modules/govuk/manifests/nodes/s_postgresql_db_admin_base.pp

```diff
@@ -0,0 +1,79 @@
+# == Class: govuk::nodes::s_postgresql_db_admin_base
+#
+# This machine class defines the common configuration used to
+# administer the app-specific DB-admin machines for PostgreSQL
+# RDS instances.
+#
+# === Parameters
+#
+# [*postgres_host*]
+#   Hostname of the RDS database to use.
+#   Default: undef
+#
+# [*postgres_user*]
+#   The PostgreSQL user to use for admisistering the database.
+#   Default: undef
+#
+# [*postgres_password*]
+#   The password corresponding to the above `postgres_user`.
+#   Default: undef
+#
+# [*postgres_port*]
+#   The port with which to connect to the `postgres_host`.
+#   Default: '5432'
+#
```

Comment:
Oh should we have apt_mirror_hostname documented too?

Reply:
Sure 👍 I've taken the description from https://github.com/alphagov/govuk-puppet/blob/e8c541a58ec0cec12b781688eb77b28e50220ecc/modules/govuk_jenkins/manifests/package.pp#L8-L9.